forest/src/cli/wallet_cmd.rs

```diff
@@ -165,10 +166,28 @@ impl WalletCommands {
             }
             Self::List => {
                 let response = wallet_list().await.map_err(handle_rpc_err).unwrap();
+                let default = wallet_default_address()
+                    .await
+                    .map_err(handle_rpc_err)
+                    .unwrap();
 
-                response.iter().for_each(|address| {
-                    println!("{}", address.0);
-                });
+                println!("Address                                    Default  Balance");
```

Comment:
@jimluo let's add the same explicit padding like for the data row

Reply:
Thanks, I will fix it over the weekend